### DISCUSSION TITLE
how is it financed

### DISCUSSION THREAD

**Discussion by eguon46** [iSoron/uhabits#1485] [Help & FAQ]
How is the app financed? 
It's free and donations are not possible. Leaves me with the conclusion it sells user data. 
Does anyone have any informations? 

Thanks for the replies

**Comment by LordJon** [comment]
That's always something to be concerned about. If in doubt you can always check the source code.

However not receiving donations nor the fact that it's free does automatically mean they sell the data. If that was a logical conclusion then there's a lot of software out there that does (most but not all open source software is free).

**Comment by iSoron** [comment]
Hi @eguon46, as explained in our privacy policy, we don't collect or sell user data. The app is not financed; we have no source of revenue. All development is done by volunteers in their free time.
